packages/wonder-blocks-core/components/clickable-behavior.js

```diff
@@ -89,7 +89,9 @@ type Props = {|
     role?: ClickableRole,
 |};
 
-type State = {|
+type State = ClickableState;
```

Comment:
oh, my point here is that you don't need this line at all. Instead of it, you can use `ClickableState` in the component definition. More exactly here: https://github.com/Khan/wonder-blocks/blob/a250b383272bbb94070f2dff224ebfcc4151a449/packages/wonder-blocks-core/components/clickable-behavior.js#L240

Reply:
I see, I fixed it in the latest commit